### PR TITLE
MODFQMMGR-111: Add array-based fields back into entity type definitions

### PR DESCRIPTION
**Pull request by bvsharp**
## Purpose
Add array-based fields back into entity type definitions. Some code changes are also required to keep the fields working with the new dynamic query logic.

## Testing
- [x] All unit tests passed
- [x] Array-based fields can be queried
- [x] Array-based fields show up correctly in query results
- [x] UserFriendlyQueryService works correctly with new fields

### CODE REVIEW

File: src/test/java/org/folio/fqm/repository/QueryRepositoryTest.java [hunk 1]
```diff
@@ -12,7 +12,6 @@
 
 import java.time.Duration;
 import java.time.OffsetDateTime;
-import java.time.temporal.ChronoUnit;
```
Comment (bvsharp) [inline src/test/java/org/folio/fqm/repository/QueryRepositoryTest.java:15]: Removed this unused import just to change some code and get Jenkins to trigger another build

File: translations/mod-fqm-manager/en_GB.json [hunk 2]
```diff
@@ -143,9 +143,17 @@
     "entityType.drv_item_holdingsrecord_instance.item_updated_date": "Item updated date",
     "entityType.drv_user_details": "Users",
     "entityType.drv_user_details.user_active": "User active",
+    "entityType.drv_user_details.user_address_ids": "User address IDs",
```
Comment (kjain110) [inline translations/mod-fqm-manager/en_GB.json:146]: @bvsharp  I guess we only need to change en.json. No other files needed I guess

File: translations/mod-fqm-manager/en_SE.json [hunk 3]
```diff
@@ -143,9 +143,17 @@
     "entityType.drv_item_holdingsrecord_instance.item_updated_date": "Item updated date",
     "entityType.drv_user_details": "Users",
     "entityType.drv_user_details.user_active": "User active",
+    "entityType.drv_user_details.user_address_ids": "User address IDs",
```
Comment (kjain110) [inline translations/mod-fqm-manager/en_SE.json:146]: @bvsharp I guess we only need to change en.json. No other files needed I guess

File: translations/mod-fqm-manager/en_US.json [hunk 4]
```diff
@@ -143,9 +143,17 @@
     "entityType.drv_item_holdingsrecord_instance.item_updated_date": "Item updated date",
     "entityType.drv_user_details": "Users",
     "entityType.drv_user_details.user_active": "User active",
+    "entityType.drv_user_details.user_address_ids": "User address IDs",
```
Comment (kjain110) [inline translations/mod-fqm-manager/en_US.json:146]: same as above

**Review by mweaver-ebsco — APPROVED** [review]

**Review by kjain110 — APPROVED** [review]

**Review by ncovercash — APPROVED** [review]

**Pull request merged**